ClassDemo/ClassDemo.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+namespace ClassDemo.Tests;
+
+public class UnitTest1
+{
+    [Fact]
+    public void Test1()
+    {
+
+    }

```

Comment:
```suggestion
    public void Test1()
    {
       Assert.Equal(1,2);
    }

```
I think you want this really good assert